codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -2258,25 +2259,28 @@ wrappedEvent ::
     -> Event
     -> Wrapped a
     -> Wrapped a
-wrappedEvent _ _ (MouseMovement _) w = w {mouseMovedTime = 0}
-wrappedEvent ctrls f (MousePress LeftButton p) w =
-    (foldr (handleControl f p) w (ctrls w)) {mouseMovedTime = 0}
-wrappedEvent _ _ _ w = w
+wrappedEvent _ _ (TimePassing _) w = w
+wrappedEvent ctrls f (event) w =
+    (foldr (handleControl f (event)) w (ctrls w)) {mouseMovedTime = 0}
```

Comment:
There's no need for parentheses around `event`.  Parentheses around a single variable are always unneeded.